app/townsquare/management/commands/payout_mini_clr.py

```diff
@@ -62,7 +62,7 @@ def handle(self, *args, **options):
         network = 'mainnet' if not settings.DEBUG else 'rinkeby'
         from_address = settings.MINICLR_ADDRESS
         from_pk = settings.MINICLR_PRIVATE_KEY
-        DAI_ADDRESS = '0x6b175474e89094c44da98b954eedeac495271d0f' if network=='mainnet' else '0x6A9865aDE2B6207dAAC49f8bCba9705dEB0B0e6D'
+        DAI_ADDRESS = '0x6b175474e89094c44da98b954eedeac495271d0f' if network=='mainnet' else '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa'
```

Comment:
^ Was this update intended ?

Reply:
Yea this was intended.

The first address `0x6A9865aDE2B6207dAAC49f8bCba9705dEB0B0e6D` is the official Rinkeby DAI address. But Oasis on Rinkeby has been down for a while now, so it's very difficult to mint DAI on RInkeby.

We switched to `0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa`, which is the address for Rinkeby DAI given out when using the faucet available on https://app.compound.finance/. This makes it a lot easier to get Rinkeby DAI for development/testing